compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/tile_copy_using_scf_for.mlir

```diff
@@ -0,0 +1,61 @@
+// RUN: iree-opt --pass-pipeline='builtin.module(func.func(iree-amdaie-tile-and-fuse{use-scf-for tiling-level=2 target-op=2}))' --split-input-file %s | FileCheck %s --check-prefix=CHECK-RHS-COPY
+// RUN: iree-opt --pass-pipeline='builtin.module(func.func(iree-amdaie-tile-and-fuse{use-scf-for tiling-level=1 target-op=3}))' --split-input-file %s | FileCheck %s --check-prefix=CHECK-LHS-COPY
+
+#map = affine_map<(d0) -> (d0 * 64)>
+#config = #iree_codegen.lowering_config<tile_sizes = [[64, 64], [0, 256], [256, 0]]>
+func.func @matmul_example_dispatch_0_matmul_2048x2048x2048_i32() {
+  %c0_i32 = arith.constant 0 : i32
+  %c0 = arith.constant 0 : index
+  %0 = hal.interface.binding.subspan set(0) binding(0) type(storage_buffer) alignment(64) offset(%c0) flags(ReadOnly) : !flow.dispatch.tensor<readonly:tensor<2048x2048xi32>>
+  %1 = hal.interface.binding.subspan set(0) binding(1) type(storage_buffer) alignment(64) offset(%c0) flags(ReadOnly) : !flow.dispatch.tensor<readonly:tensor<2048x2048xi32>>
+  %2 = hal.interface.binding.subspan set(0) binding(2) type(storage_buffer) alignment(64) offset(%c0) : !flow.dispatch.tensor<writeonly:tensor<2048x2048xi32>>
```

Comment:
The test case can be made concise - for instance, you can get rid of all the `hal` constructs.